appland/webview.js

```diff
@@ -38,8 +38,13 @@ export function mountWebview() {
 
   messages.on('init', ({data: initData}) => {
     // messages emitted by the webview
-    app.$on('viewSource', (location) => vscode.postMessage({command: 'view-source', location}))
+    app.$on('viewSource', (location) => vscode.postMessage({command: 'viewSource', location}))
     app.$on('uploadAppmap', () => vscode.postMessage({command: 'uploadAppMap'}))
+    app.$on('sidebarSearchFocused', () => vscode.postMessage({command: 'sidebarSearchFocused'}))
```

Comment:
The alternative to the telemetry-related messages would be a message like `telemetryMessage` with the properties embedded. But that would move telemetry handling into the JS source...
If there's going to be a lot more messages like these, then I'd probably move it here to avoid the verbose handling of telemetry.